docs/apache-airflow/migrations-ref.rst

```diff
@@ -20,218 +20,242 @@ Reference for Database Migrations
-+--------------------------------+------------------+-----------------+---------------------------------------------------------------------------------------+
-| ``2e541a1dcfed``               | ``1b38cef5b76e`` | ``1.6.0``       | Change ``task_instance.task_duration`` type to ``FLOAT``                              |
-+--------------------------------+------------------+-----------------+---------------------------------------------------------------------------------------+
-| ``1b38cef5b76e``               | ``502898887f84`` | ``1.6.0``       | Add ``dag_run`` table                                                                 |
-+--------------------------------+------------------+-----------------+---------------------------------------------------------------------------------------+
-| ``502898887f84``               | ``52d714495f0``  | ``1.6.0``       | Adding ``extra`` column to ``Log`` table                                              |
-+--------------------------------+------------------+-----------------+---------------------------------------------------------------------------------------+
-| ``52d714495f0``                | ``338e90f54d61`` | ``1.5.2``       | Add indices in ``job`` table                                                          |
-+--------------------------------+------------------+-----------------+---------------------------------------------------------------------------------------+
-| ``338e90f54d61``               | ``13eb55f81627`` | ``1.5.0``       | Add ``operator`` and ``queued_dttm`` to ``task_instance`` table                       |
-+--------------------------------+------------------+-----------------+---------------------------------------------------------------------------------------+
-| ``13eb55f81627``               | ``1507a7289a2f`` | ``1.5.0``       | Maintain history for compatibility with earlier migrations                            |
-+--------------------------------+------------------+-----------------+---------------------------------------------------------------------------------------+
-| ``1507a7289a2f``               | ``e3a246e0dc1``  | ``1.5.0``       | Add ``is_encrypted`` column in ``connection`` table                                   |
-+--------------------------------+------------------+-----------------+---------------------------------------------------------------------------------------+
-| ``e3a246e0dc1``                |                  | ``1.5.0``       | Create initial schema                                                                 |
-+--------------------------------+------------------+-----------------+---------------------------------------------------------------------------------------+
+
+ .. This table is automatically updated by pre-commit by ``scripts/ci/pre_commit/pre_commit_migration_documented.py``
+ .. Changes should be made to config file ``airflow/migrations/migrations-info.yaml``
+ .. Beginning of auto-generated table
+
```

Comment:
one reason the diff is ugly is i wrap a little more aggressively

but also the column widths end up being a little different when using `tabulate` and wasn't having luck getting it to conform so left it with the ugly diff